cidc_api/models/files/facets.py

```diff
@@ -204,17 +219,18 @@ def __init__(self, match_clauses: List[str], description: Optional[str] = None):
         ),
     },
     "Olink": {
-        "All Olink Files": FacetConfig(
+        "Run Level": FacetConfig(
```

Comment:
Maybe hyphenate? `Run-Level` and `Study-Level`.

Reply:
in next commit